### ISSUE TITLE
Improve typing of sync-v2 status API

### ISSUE THREAD

**Issue by glevco** [HathorNetwork/hathor-core#727]
Should we create a `NamedTuple` to encapsulate this return type?

_Originally posted by @glevco in https://github.com/HathorNetwork/hathor-core/pull/275#discussion_r1271117402_

**Comment by jansegre** [comment]
Makes sense. I think there's also "typed dicts" in Python that we could consider too.
